### PR TITLE
tutorial -developer app guide

### PR DESCRIPTION
**Pull request by hm90121**

### CODE REVIEW

File: src/docs/_sidebar.md [hunk 1]
```diff
@@ -238,13 +238,13 @@
 
   <!-- - [Customizing Engine Input](/tutorials/pages/customizing-engine-input.md) -->
 
-  <!-- - [Developer App Guide](/tutorials/pages/temp-under-construction.md) -->
+   - [Developer App Guide](/tutorials/developer-app/) -->
```
Comment (kevmurray) [inline src/docs/_sidebar.md:241]: remove end comment

Comment (hm90121) [reply src/docs/_sidebar.md]: ok

File: src/docs/tutorials/pages/developer-app/README.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+
+![Engines](engines.png)
+
+For every Engine listed, their capabilities, type, last updated, and status(active, pending) are also specified. To know which capabilities each Engine can support, check our [Capabilities Table](https://docs.veritone.com/#/developer/engines/cognitive/?id=capabilities). 
+
+You can also get build information, logs, and how many tasks are running on an engine, etc., by choosing the specific one on the list. For example, we will check additional information about "Face Detection - F V3" engine.
+
+Clicking on the "Face Detection - F V3" engine will redirect you to a new page where you will see the unique engine id, option to generate manifests for docker builds, see build information, tasks, and logs.See the screenshot below.
+
+![Engine Overview](engine-overview.png)
```
Comment (kevmurray) [inline src/docs/tutorials/pages/developer-app/README.md:138]: I see that these references have been changed to lowercase extensions (.png), but the files still seem to be uppercase (.PNG) - does that work?

No, it does not seem to.

Please rename all the images with lowercase extensions. Note that sometimes changing case doesn't work correctly, especially on a Mac. In order to change just the case of a file, you may need to delete them and re-add them.

To do this:
1. Move all the .PNG files to a backup directory
2. Do a commit where you delete all the .PNG files
3. Copy the files from the backup directory to where they need to be
4. Rename them all with .png
5. Do a second commit where you add them back in

**Review by kevmurray — CHANGES_REQUESTED** [review]

File: src/docs/tutorials/pages/developer-app/README.md (hunk 2 above)
Comment (hm90121) [reply src/docs/tutorials/pages/developer-app/README.md:138]: i have kept the files .png during commit idont know why they are automatically changing to uppercase when uploaded

**Pull request closed without merging**